app/views/common/_domain.html.erb

```diff
@@ -1,11 +0,0 @@
-<%= fields_for item do |f| %>
-  <%= select_f f, :subnet_id, domain_subnets, :id, :to_label,
-               { :include_blank => domain_subnets.any? ? blank_or_inherit_f(f, :subnet) : _("No subnets") },
-               { :disabled => domain_subnets.empty?,
-                 :data     => { :url     => SETTINGS[:unattended] ? freeip_subnets_path : '',
-                                :subnets => subnets_ipam_data.to_json },
-                 :label    => _("Subnet"),
-                 :help_inline => :indicator,
-                 :onchange => 'subnet_selected(this);' }
```

Comment:
If I understand correctly, `interface_subnet_selected` is only relevant for interfaces, not hostgroups (autoselect IP etc.). Am I missing something here?